extensions/podman/packages/extension/src/extension.ts

```diff
@@ -1420,19 +1420,6 @@ export async function activate(extensionContext: extensionApi.ExtensionContext):
     const compatibilityEnabled = socketCompatibilityMode.isEnabled();
     extensionApi.context.setValue(PODMAN_DOCKER_COMPAT_ENABLE_KEY, compatibilityEnabled);
 
-    // Create a status bar item to show the status of compatibility mode as well as
-    // create a command so when you can disable / enable compatibility mode
-    const statusBarItem = extensionApi.window.createStatusBarItem();
-    statusBarItem.text = 'Docker Compatibility';
-    statusBarItem.command = 'podman.socketCompatibilityMode';
-
-    // Use tooltip text from class
-    statusBarItem.tooltip = socketCompatibilityMode.tooltipText();
-
-    statusBarItem.iconClass = 'fa fa-plug';
-    statusBarItem.show();
-    extensionContext.subscriptions.push(statusBarItem);
-
     // Create a modal dialog to ask the user if they want to enable or disable compatibility mode
     const command = extensionApi.commands.registerCommand('podman.socketCompatibilityMode', async () => {
```

Comment:
question: if we remove the status bar entry calling the command 'podman.socketCompatibilityMode'
is that 'podman.socketCompatibilityMode' command is called from other places ? if not command should be removed

Reply:
There is Docker compatibility settings and onDidChangeConfiguration listener calls this method indirectly.

Reply:
It seems not used anywhere, I'll verify it.

Reply:
The command is still available from through command palette and triggers the same workflow. If the plan is to use this command from somewhere else it should stay. 
CC: @Firewall

Reply:
I don't think it is needed to remove it from the command palette.

Reply:
That command is still used in other places such as the Podman Mac helper notification.